security/RememberLoginHash.php

```diff
@@ -0,0 +1,166 @@
+<?php
+/**
+ * Persists a token associated with a device for users who opted for the "Remember Me"
+ * feature when logging in.
+ * By default, logging out will discard all existing tokens for this user
+ * The device ID is a temporary ID associated with the device when the user logged in
+ * and chose to get the login state remembered on this device. When logging out, the ID 
+ * is discarded as well.
+ *
+ * @package framework
+ * @subpackage security
+ *
+ * @property string $DeviceID
+ * @property string $RememberLoginHash
+ */
+
+class RememberLoginHash extends DataObject {
+
+	private static $db = array (
+		'DeviceID' => 'Varchar(40)',
+		'RememberLoginHash' => 'Varchar(160)',
```

Comment:
The naming is a bit verbose, same as class name - can we make it `Hash` or `Value`?